pkg/reconciler/trigger/trigger.go

```diff
@@ -236,6 +252,71 @@ func (r *Reconciler) reconcileRetryTopicAndSubscription(ctx context.Context, tri
 	return nil
 }
 
+// getPubsubRetryPolicy gets the eventing retry policy from the Broker delivery
+// spec and translates it to a pubsub retry policy.
+func getPubsubRetryPolicy(spec *eventingduckv1beta1.DeliverySpec) (*pubsub.RetryPolicy, error) {
+	if spec == nil || spec.BackoffDelay == nil || spec.BackoffPolicy == nil {
+		return nil, nil
+	}
+	// The Broker delivery spec is translated to a pubsub retry policy in the
+	// manner defined in the following post:
+	// https://github.com/google/knative-gcp/issues/1392#issuecomment-655617873
```

Comment:
This should probably be documented somewhere in the docs/ directory as well.

Reply:
I added a docs file to explain this: https://github.com/google/knative-gcp/pull/1536/files#diff-86642e88be266e549139dbe5b09c2d8d